apps/discoverbos/widget/project/module.jsx

```diff
@@ -0,0 +1,112 @@
+function Card({ metadata, path }) {
```

Comment:
I moved the card to a shared module. Modules make the load time faster since the code is cached and reused for each feed item rather than us rendering a new widget

Reply:
I didn't knew that, looks good